app/assets/templates/aspect_create_modal_tpl.jst.hbs

```diff
@@ -1,40 +1,51 @@
-<div class="modal hide fade" id="newAspectModal" tabindex="-1" role="dialog" aria-labelledby="newAspectModalLabel" aria-hidden="true">
+<div class="modal fade" id="newAspectModal" tabindex="-1" role="dialog" aria-labelledby="newAspectModalLabel"
+     aria-hidden="true" xmlns="http://www.w3.org/1999/html">
 
-  <div class="modal-header">
-    <button class="close" type="button" data-dismiss="modal" aria-hidden="true">&times;</button>
-    <h3 id="newAspectModalLabel">{{ t "aspects.create.add_a_new_aspect" }}</h3>
-  </div>
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button class="close" type="button" data-dismiss="modal" aria-hidden="true">&times;</button>
+        <h3 class="modal-title" id="newAspectModalLabel">{{ t "aspects.create.add_a_new_aspect" }}</h3>
```

Comment:
Actually you're right, it's not translated for a strange reason, but that's already true on develop. Please open a separate issue.